Error in loadNamespace(name) : there is no package called ‘dotenv’

Tentando criar uma imagem pra seguir com https://github.com/splor-mg/volumes-loa/issues/33 recebi o erro abaixo que definitivamente não deveria estar acontecendo porque a imagem base (`rocker/verse:3.6.3`) deveria ser versionada 

```
=> [2/8] WORKDIR /home/rstudio                                                                                       0.4s
 => [3/8] COPY texmf /opt/texmf-local                                                                                 0.1s
 => [4/8] RUN texhash                                                                                                 1.4s
 => [5/8] RUN Rscript -e "install.packages('dotenv')"                                                                 5.2s
 => ERROR [6/8] RUN --mount=type=secret,id=secret Rscript -e     "dotenv::load_dot_env('/run/secrets/secret'); remot  1.2s
------
 > [6/8] RUN --mount=type=secret,id=secret Rscript -e     "dotenv::load_dot_env('/run/secrets/secret'); remotes::install_bitbucket('dcgf/relatorios@v0.6.75', auth_user = Sys.getenv('BITBUCKET_AUTH_USER'), password = Sys.getenv('BITBUCKET_APP_PASSWORD'))":
#11 1.119 Error in loadNamespace(name) : there is no package called ‘dotenv’
#11 1.119 Calls: :: ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
#11 1.120 Execution halted
------
executor failed running [/bin/sh -c Rscript -e     "dotenv::load_dot_env('/run/secrets/secret'); remotes::install_bitbucket('dcgf/relatorios@$relatorios_version', auth_user = Sys.getenv('BITBUCKET_AUTH_USER'), password = Sys.getenv('BITBUCKET_APP_PASSWORD'))"]: exit code: 1
make: *** [build] Error 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -5,7 +5,7 @@ ARG reest_version
 WORKDIR /home/rstudio
 COPY texmf /opt/texmf-local
 RUN texhash
-RUN Rscript -e "install.packages('dotenv')"
+RUN Rscript -e "install.packages('dotenv', repos = 'https://packagemanager.posit.co/cran/2020-04-24/')"
 RUN --mount=type=secret,id=secret Rscript -e \
     "dotenv::load_dot_env('/run/secrets/secret'); remotes::install_bitbucket('dcgf/relatorios@$relatorios_version', auth_user = Sys.getenv('BITBUCKET_AUTH_USER'), password = Sys.getenv('BITBUCKET_APP_PASSWORD'))"
 RUN --mount=type=secret,id=secret Rscript -e \

